packages/@webex/plugin-meetings/src/meeting/util.ts

```diff
@@ -202,7 +202,11 @@ const MeetingUtil = {
         meeting.reconnectionManager.cleanUp();
       })
       .then(() => meeting.stopKeepAlive())
-      .then(() => meeting.updateLLMConnection());
+      .then(() => {
+        if (meeting.config?.enableAutomaticLLM) {
+          meeting.updateLLMConnection();
```

Comment:
is the `return` here missing on purpose?

Reply:
@Kesari3008 can you answer the question above?

Reply:
@marcin-bazyl There has been no return statement present here from the start. This is the cleanup function where we needed to added this check to avoid updating LLM connection when it's not enabled.
You can refer to PR: https://github.com/webex/webex-js-sdk/pull/3479 where this change has gone in for more details.

Reply:
This was the wrong approach. updateLLMConnection was designed to be idempotent, so you could call it from wherever and it wouldn't be a problem. enableAutomaticLLM was not the only time where LLM could be enabled. The intention was to be able to manually enable LLM later on in the meeting flow, in which case a cleanup would still be required. That being said, it isn't a large enough issue to want to fix in this PR.